Cubical/Categories/Functor/Properties.agda

```diff
@@ -0,0 +1,124 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+
+module Cubical.Categories.Functor.Properties where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.GroupoidLaws using (lUnit; rUnit; assoc; cong-∙)
+open import Cubical.Categories.Category
+open import Cubical.Categories.Functor.Base
+
+private
+  variable
+    ℓC ℓC' ℓD ℓD' ℓE ℓE' : Level
+    ℓ ℓ' ℓ'' : Level
+    B : Precategory ℓC ℓC'
+    C : Precategory ℓC ℓC'
+    D : Precategory ℓD ℓD'
+    E : Precategory ℓE ℓE'
+
+open Precategory
+open Functor
+
+infixr 15 _◍_
+-- is there actual function composition in the library somewhere?
+_◍_ : ∀ {X : Type ℓ} {Y : Type ℓ'} {Z : Type ℓ''} → (Y → Z) → (X → Y) → (X → Z)
+(g ◍ f) x = g (f x)
+
```

Comment:
As opposed to the dependent one in `Cubical.Foundations.Function` you mean?

Reply:
Ah I guess I didn't do enough digging, thanks!